src/Artsy/Router/RouterLink.tsx

```diff
@@ -19,15 +20,38 @@ export const RouterLink: React.FC<LinkProps> = (
 ) => {
   const isRouterContext = Boolean(context.router)
 
+  // Only pass found-router specific props across
```

Comment:
I think this comment is either incorrect, or could use more definition of what is a found-specific prop. Very minor, but if I come back here in the future, I think there's a mismatch between what I'd read in the comment and which props are being passed through.